.github/workflows/deploy-fleet-website.yml

```diff
@@ -62,6 +62,9 @@ jobs:
     # > script here.  For more info, see: https://github.com/fleetdm/fleet/pull/5549
     - run: rm -f .gitignore
 
+    # build storybook in the website's assets/ folder
+    - run: npm run build-storybook -- -o ./website/assets/storybook
```

Comment:
two thoughts:

1. should we set `--loglevel verbose` to ease troubleshooting?
2. should we add this directory to `.gitignore`? I see we're deleting `.gitignore` in CI and adding some files with `git add -f` so they can be sent to Heroku, so if it might not be worth the trouble.

Reply:
> should we set --loglevel verbose to ease troubleshooting?

Makes sense to me! I just made a commit with that change.

> should we add this directory to .gitignore? I see we're deleting .gitignore in CI and adding some files with git add -f so they can be sent to Heroku, so if it might not be worth the trouble.

I don't think we need to do this. We're only running this command when deploying the website, and the built storybook folder will only exist in the Heroku app.

Reply:
sounds good!